go-controller/pkg/libovsdb/ops/chassis.go

```diff
@@ -151,10 +151,10 @@ func CreateOrUpdateChassis(sbClient libovsdbclient.Client, chassis *sbdb.Chassis
 			BulkOp:         false,
 		},
 		{
-			Model:          chassis,
-			OnModelUpdates: onModelUpdatesAllNonDefault(),
-			ErrNotFound:    false,
-			BulkOp:         false,
+			Model:            chassis,
+			OnModelMutations: []interface{}{&chassis.OtherConfig, &chassis.Encaps},
```

Comment:
I must confess how I never liked how CMS and ovn-controller messes with these fields. I like better when we operated in a world where cms messes with external-idsa nd other-confog was only mnanaged by ovn-controller:
https://github.com/ovn-org/ovn-kubernetes/pull/3169#discussion_r1162003141  <-- Reference link